### DISCUSSION TITLE
Files saved on s3 compatible like Storj not displaying

### DISCUSSION THREAD

**Discussion by kofyebs** [statamic/cms#5814] [Q&A]
I have setup all the settings.
I followed the instructions in [this](https://statamic.dev/tips/digital-ocean-spaces-for-asset-container) 
Below is what i have in the `.env`
`DO_STORJ_KEY=xxxxx`
`DO_STORJ_SECRET=xxxxx`
`DO_STORJ_ENDPOINT=https://gateway.eu1.storjshare.io`
`DO_STORJ_REGION=gateway.eu1`
`DO_STORJ_BUCKET=statamic`
`DO_STORJ_ROOT=/`
`DO_STORJ_URL=https://example.com/cdn/`
I have already uploaded some files beforehand and i am able to list all the files in my existing bucket inside the asset container in the CP. But when i select a file from the asset container to be used as featured audio in an article, 
the audio does not play so i tried inspecting the source code and i can this,
`<audio id="jp_audio_1" preload="metadata" src="https://example.com/audio/demo.mp3" title="Demo"></audio>`
also trying to display image from the bucket, i inspected the source code i found this and its also not showing
`<figure><img src="/img/asset/c3RvcmovYXJ0d29ya3MvZHVtbXkuanBn?w=265&amp;h=265&amp;s=c510fa0ebd7a0789893f0c4803bce1b2" alt=""></figure>`

**Comment by edalzell** [comment]
That second url is a glide url, but you're talking about an mp3 file?

**Reply by kofyebs** [reply]
I have both files, image and mp3 already uploaded to Storj

**Comment by edalzell** [comment]
and this is a local url, not a Storj one: https://example.com/audio/demo.mp3 ?

**Reply by kofyebs** [reply]
Yes, but i don't know how to get the Storj url to make it work.

**Reply by jasonvarga** [reply]
I've never used or even heard of Storj, but if you upload an image and then view it in the Storj dashboard... the URL would be whatever is before your image path. eg. `http://bucket.storj.com/your/file/here.jpg` it would be `http://bucket.storj.com`
